### PR TITLE
CEDS-2499 Schema update

### PR DESCRIPTION
**Pull request by PauloPortugal**
* Added Communication to Carrier
* Removed Manufacturer and RefundRecipientParty
* Sequence Numeric is now mandatory on WriteOffQuantityQuantityType

### CODE REVIEW

File: pointers/data/temp/raw_pointers [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (PatrykRudnicki) [inline pointers/data/temp/raw_pointers]: Is this file necessary?

**Review by PatrykRudnicki — APPROVED** [review]

File: pointers/data/temp/raw_pointers (hunk 1 above)
Comment (PauloPortugal) [reply pointers/data/temp/raw_pointers]: No. I have fat fingered that one.

Comment (MaciejRewera) [inline pointers/data/temp/raw_pointers]: ???

Comment (PauloPortugal) [reply pointers/data/temp/raw_pointers]: This has been addressed.

Comment (PauloPortugal) [reply pointers/data/temp/raw_pointers]: Ignore. that was by mistake. Another work was trying to sneak in.

File: src/main/java/wco/datamodel/wco/res_dms/_2/Response.java [hunk 2]
```diff
@@ -360,8 +360,8 @@
      * <p>
      * You are getting this "catch-all" property because of the following reason: 
      * The field name "Declaration" is used by two different parts of a schema. See: 
-     * line 1426 of file:/Users/developer/workspace/hmrc/wco-dec/updateSchema/customs-declarations/public/api/conf/1.0/schemas/wco/notification/WCO_RES_2_DMS.xsd
-     * line 832 of file:/Users/developer/workspace/hmrc/wco-dec/updateSchema/customs-declarations/public/api/conf/1.0/schemas/wco/notification/WCO_RES_2_DMS.xsd
+     * line 1426 of file:/home/pmonteiro-work/Workspace/wco-dec/updateSchema/customs-declarations/public/api/conf/1.0/schemas/wco/notification/WCO_RES_2_DMS.xsd
+     * line 832 of file:/home/pmonteiro-work/Workspace/wco-dec/updateSchema/customs-declarations/public/api/conf/1.0/schemas/wco/notification/WCO_RES_2_DMS.xsd
```
Comment (MaciejRewera) [inline src/main/java/wco/datamodel/wco/res_dms/_2/Response.java]: Nit: It's probably automaticaly generated but relative path should be enough. Not to expose your own filesystem.

**Review by MaciejRewera — APPROVED** [review]
LGTM

File: src/main/java/wco/datamodel/wco/res_dms/_2/Response.java (hunk 2 above)
Comment (PauloPortugal) [reply src/main/java/wco/datamodel/wco/res_dms/_2/Response.java]: Ahhh good point. Let me remove that.
Done.
